tests/lit/multi/extern_C_fb_init/main.st

```diff
@@ -0,0 +1,7 @@
+FUNCTION main 
```

Comment:
Can you add a similar test in codegen?
I would like to see the case where we have a function using a function block, and that functionblock has a user_init be tested.
I expect the function, while initializing the function block call to (implicitly) call fb_init. I'm fine with adding this as a followup if we know the case works, but it would be good to express this in codegen tests

Reply:
I'll add a test for this in #1471